src/main/java/cc/bitbank/entity/Transactions.java

```diff
@@ -38,4 +58,8 @@ public Transactions () {}
     public Transactions (Transaction[] transactions) {
         this.transactions = transactions;
     }
+
+    public Transaction[] getTransactions() {
```

Comment:
Please remove along with all the other new methods. (this should be a separate pull request)